src/main/app/src/components/toteutus/ToteutusHakukohteet.tsx

```diff
@@ -97,7 +104,22 @@ const HakuCardGrid = ({ tyyppiOtsikko, icon, toteutus, hakukohteet }: GridProps)
   const { osoitteet } = useOsoitteet(oppilaitosOids, true);
 
   const { isDraft } = useUrlParams();
-
+  function getFullToimipisteNimi(oid: string) {
+    const toimipiste = find(oppilaitosOsat, function (opOsa) {
+      if (get(opOsa, 'oid', 'toimipiste oid not found') === oid) return true;
```

Comment:
Voiskohan näitä yksinkertaistaa vähän
```suggestion
      return get(opOsa, 'oid', 'toimipiste oid not found') === oid
```